pkgs/by-name/hi/hiddify-next/package.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, fetchFromGitHub
+, pkg-config
+, flutter
+, buildGoModule
+, libayatana-appindicator
+}:
+let
+  pname = "hiddify-next";
+  version = "1.5.2";
+
+  hiddify = fetchFromGitHub {
```

Comment:
One may want to make these packages overridable. There are some solutions:

1. use `callPackage` to load from another file, as a default argument `{ ..., hiddify? callPackage ./hiddify.nix {}, ... }: flutter... { ... }`, and expose them in `passthru` of the main package
2. use `callPackages`, and write a set of packages, like https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/tools/language-servers/nixd/default.nix
3. just split them into different by-name directories, and inherit `version` and `src`.